src/test/java/com/snowflake/kafka/connector/records/ConverterTest.java

```diff
@@ -57,6 +68,27 @@
 
   private static final String TEST_TOPIC = "test";
 
+  // For each task, we require a separate instance of SimpleDataFormat, since they are not
+  // inherently thread safe
+  private static final ThreadLocal<SimpleDateFormat> ISO_DATE_FORMAT_THREAD_SAFE =
```

Comment:
Instead of re-declaring, can you make the record service member public static (looks like it was public static before) and use it in the test?

Reply:
Ahh yes. Will do..